src/energyplus/ForwardTranslator.cpp

```diff
@@ -3073,6 +3083,7 @@ namespace energyplus {
   }
 
   std::vector<IddObjectType> ForwardTranslator::iddObjectsToTranslate() {
+    // TODO: avoid the call to iddObjectsToTranslateInitializer with the ton of push_backs
```

Comment:
This is a great performance catch. Something needed to be done.

I might be off base here, but this has a bit of a code smell to my nose. it does the job sure. but maybe we can use a member variable and make use of member initialization to do this job. local statics are questionable to me.